src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -33,6 +33,12 @@
                 <span>{{ collector.tokenClass|abbr_class }}</span>
             </div>
             {% endif %}
+            {% if collector.firewall|default([]) is not empty %}
+            <div class="sf-toolbar-info-piece">
+                <b>Firewall name</b>
+                <span>{{ collector.firewall.name|default('n/a') }}</span>
```

Comment:
Not having the firewall name is not a case likely to happen according to the way `FirewallConfig` is built, right ?
Anyway, some lines below, you don't apply a default filter. This should be consistent.